src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  margin: 0;
+  padding: 20px;
+  background-color: black;
+}
+
+div {
+  border: 1px solid #e75551;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
Added classes instead of tag names